### DISCUSSION TITLE
Need help to keep window on top of other window(multiple window) in Winui3.

### DISCUSSION THREAD

**Discussion by Sunilam** [microsoft/WindowsAppSDK#1847] [Q&A] [answered]
HI

Could you please help me to keep the window on top of the other window.(like multiple window)
I need my UI should looks like below screen shot.
1st window should be transparent
2nd window should appear on top of 1st window.
<img width="576" alt="Winui3window" src="https://user-images.githubusercontent.com/7436855/143550529-c714a83a-c02d-486d-b0d7-d89d457fda50.png">

**Comment by marb2000** [comment, marked as answer]
Hi @Sunilam,
Unfortunately, your scenario is not achievable, WindowsAppSDK 1.0 doesn't support multiple windows. It will come in the next release 1.1. Also, XAML Window objects don't support background transparency either.
